### ISSUE TITLE
Is it possible to make the timeline highlight the events which we're selecting above?

### ISSUE THREAD

**Issue by unphased** [KDAB/hotspot#307]
**Is your feature request related to a problem? Please describe.**
It's not possible to see where in the timelines a given symbol corresponds to

**Describe the solution you'd like**
We can see by filtering the timeline what symbols were hit at what time. It would be awesome if the reverse association could be shown, this can be done by highlighting the events in the timeline whose stacks intersect the symbol that we select or hover in the top section. 

As an example, I have a function that does image processing work. I want to know whether its calls are properly distributed across the multiple threads in my pools. This feature would show them all instantly, whereas the workaround is a lot of manual work for imperfect information. We do have to use copious amounts of trial and error to narrow down even just in which thread a particular bit of code executed.

**Describe alternatives you've considered**
I know that each event shown is associated with a stack, and so various information can be shown about it when I filter by thread and time, this works even down to a single event. So the workaround is to spend a lot of time hunting around CPUs/threads and guess where I think my method is being called, and doing a sort of binary search hunting around, knowing that if I select threads and time regions where the symbol i'm referring to is not shown anymore, then it cannot be in the zone I've filtered. Then, rinse and repeat.

**Comment by milianw** [comment]
This is implemented already, when you hover a symbol, the corresponding events are highlighted in the timeline.

What version of hotspot are you using?

**Comment by unphased** [comment]
Thanks! I'll check it again then. I don't recall the version but it was just a week ago and I did test a recently downloaded AppImage (from within a docker container).

**Comment by unphased** [comment]
I'm testing with `hotspot-v1.3.0-x86_64.AppImage`. The only thing that seems to do anything based on hover is the flamegraph. It does not highlight any timeline events that i can tell, but I did discover that I can click and Filter In on a given symbol and that narrows it down in the timeline. However it's not as ideal as what I suggested (which you said is already implemented?) where the timelines stay put and just highlight while showing the rest. 

When it filters the timeline it drastically changes the listing of threads in the timeline, and prevents a qualitative understanding of the timing of these events compared to the rest.
